lib/helper/firefoxProfileCreator.py

```diff
@@ -72,7 +72,7 @@ def _install_profile_extensions(self, extensions_settings={}):
         return
 
     def remove_firefox_profile(self):
-        if os.path.isdir(self._firefox_profile_path):
+        if self._firefox_profile_path and os.path.isdir(self._firefox_profile_path):
```

Comment:
In my opinion, it's not only for speed.
The meaning of this line is simpler than only put `isdir` check here, `if we have path, and the path is a folder, then do something`. Do we have to add comment to describe that?